js/apps/admin-ui/src/realm-settings/LocalizationTab.tsx

```diff
@@ -167,21 +174,24 @@ export const LocalizationTab = ({ save, realm }: LocalizationTabProps) => {
       return { result };
     },
     ({ result }) => {
-      const bundles = Object.entries(result).slice(first, first + max + 1);
+      const bundles = Object.entries(result)
+        .map<LocaleSpecificEntry>(([key, value]) => ({ key, value }))
+        .sort((a, b) => a.key.localeCompare(b.key));
```

Comment:
We have a [`useLocaleSort()` hook](https://github.com/keycloak/keycloak/blob/a65af2d25486e323c1e5ef415f65ceb8dbdc7e79/js/apps/admin-ui/src/utils/useLocaleSort.ts#L5) that can be used to determine the current user's locale and apply the sorting aware of those preferences.

Reply:
@jonkoops I added the suggested hook. The logic should now also be more in line with other parts of the UI.